includes/REST_API/Stories_Controller.php

```diff
@@ -343,6 +343,7 @@ public function get_items( $request ) {
 			'publish' => 'publish',
 			'future'  => 'future',
 			'draft'   => 'draft',
+			'private' => 'private',
```

Comment:
Shouldn't this be part of `all` above as well?

Reply:
Can we add a check in this test for this property [here](https://github.com/google/web-stories-wp/blob/main/tests/phpunit/tests/REST_API/Stories_Controller.php#L187).

Reply:
It should not because we do not want private stories to show in the all list. It's the exception to the rule.